app/Models/ServiceTemplate.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Model;
+use LibreNMS\Interfaces\Models\Keyable;
+
+class ServiceTemplate extends Model implements Keyable
+{
+    public $timestamps = false;
+    protected $primaryKey = 'id';
+    protected $fillable = [
+        'id',
+        'device_group_id',
+        'ip',
+        'type',
+        'desc',
+        'param',
+        'ignore',
+        'status',
+        'changed',
+        'disabled',
+        'name',
+    ];
+
+    // ---- Helper Functions ----
+
+    public function getCompositeKey()
+    {
+        return $this->id . '-' . $this->device_group_id;
+    }
+
+    public static function getServiceTemplate($query)
```

Comment:
You don't need a scope function for a simple query. Use ::find($id) instead

Reply:
removed function and replaced call with ::find . Thanks!